radixdlt/src/main/java/com/radixdlt/consensus/GetVerticesResponse.java

```diff
@@ -18,31 +18,37 @@
 package com.radixdlt.consensus;
 
 import com.radixdlt.crypto.Hash;
-import java.util.function.Consumer;
+import java.util.List;
+import java.util.Objects;
 
 /**
- * An RPC request to retrieve a given vertex
+ * An RPC response
  */
-public final class GetVertexRequest {
+public final class GetVerticesResponse {
+	private final Object opaque;
```

Comment:
Why is `opaque` opaque rather than just a plain `Hash`?

Reply:
Because it contains information only relevant to the requestor for management purposes and nothing to do with the request/response itself